src/applications/disability-benefits/all-claims/pages/form0781/consentPage.js

```diff
@@ -0,0 +1,30 @@
+import {
+  radioUI,
+  radioSchema,
+} from 'platform/forms-system/src/js/web-component-patterns';
+import {
+  consentPageDescription,
+  consentPageFormHint,
+  consentPageFormTitle,
+  consentPageTitle,
+  CONSENT_OPTION_INDICATOR_CHOICES,
+} from '../../content/form0781/consentPage';
+import { formTitle } from '../../utils';
+
+export const uiSchema = {
```

Comment:
This page will also eventually need the shared [heading tag](https://www.figma.com/design/r3Aj9FtLFS989mlVeBsgJg/0781-Redesign?node-id=9442-72051&t=z8iO1AbxK4gCM22i-4) and [mental health support alert component](https://www.figma.com/design/r3Aj9FtLFS989mlVeBsgJg/0781-Redesign?node-id=9250-96482&t=7cOgvcqLHTDovLbx-4) (incorporated in https://github.com/department-of-veterans-affairs/vets-website/pull/33811)